presto-cli/src/main/java/com/facebook/presto/cli/Query.java

```diff
@@ -279,20 +283,34 @@ private void pageOutput(OutputFormat format, List<String> fieldNames)
                 });
             }
             handler.processRows(client);
+            //recordClientStop. We do this *before* we close the OutputHandler and Pager
+            recordClientStop();
         }
         catch (RuntimeException | IOException e) {
             if (client.isClientAborted() && !(e instanceof QueryAbortedException)) {
+                recordClientStop();
                 throw new QueryAbortedException(e);
             }
             throw e;
         }
     }
 
+    /**
```

Comment:
Change the comment style to //
We don't want this comment to be picked up by JavaDoc.

Reply:
It won't be - automated tools/site like [javadoc.io](https://javadoc.io/doc/com.facebook.presto/presto-cli/0.278/index.html) only pick up public methods